### ISSUE TITLE
[Bug]: Snap package doesn't follow system pref

### ISSUE THREAD

**Issue by Master811129** [sqlitebrowser/sqlitebrowser#3709]
### What did you do?

I opened the app

### What did you expect to see?

normal qt app that follows system preference with correct mouse cursor icon.

### What did you see instead?

doesn't really look like a proper modern snap application: uses the default qt theme, Fusion. 

![Screenshot_20240818_203028](https://github.com/user-attachments/assets/d80d886e-e482-4b3c-8fe6-3460bbc6e0e0)

### DB4S Version

3.13.0

### What OS are you seeing the problem on?

Linux

### OS version

Kubuntu Linux 24.04

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by FriedrichFroebel** [comment]
Somehow the versions on https://snapcraft.io/sqlitebrowser appear to be messed up: the stable version is some untagged(?) version, the candidate and beta versions point to the 3.13.0 release. Which channel are you using?

Apart from this, @deepsidhu1313 has done most of the Snap work, so he probably can help you the most about this.

**Comment by Master811129** [comment]
```
~$ snap list sqlitebrowser
Name           Version              Rev   Tracking       Publisher  Notes
sqlitebrowser  3.12.2-3569-6cde3f5  3233  latest/stable  gajjgndu   -

```
![Screenshot_20240819_014022](https://github.com/user-attachments/assets/e920dd4f-941f-49cd-9b9e-f491a6f0e10e)


Yeah its messed up. snap version in the store and the version from about section does not match.

BTW i chose 3.13.0 because It was the closest.🫢

**Comment by lucydodo** [comment]
The reason the version Snap is currently pointing to is not the latest veresion is entirely my fault. We need to update the `currentrelease` value so Snap can update the version, but for my own personal reasons I'm a little behind.

Related: https://github.com/sqlitebrowser/sqlitebrowser/discussions/3525#discussioncomment-10144276

**Comment by justinclift** [comment]
Yeah, we're likely to update the `currentrelease` values with the 3.13.1 release, which shouldn't be *too* far off. :grin:

**Comment by lucydodo** [comment]
I'll update the `currentrelease` file this weekend.
@justinclift 
I'll be touch with you about it, and apologize for the delay. As you know, I've had some personal things going on lately.

**Comment by justinclift** [comment]
No worries at all. :smile:

**Comment by lucydodo** [comment]
Notes: The `currentrelease` file has been updated.
